designer/client/component.js

```diff
@@ -35,6 +35,7 @@ export const componentTypes = {
   FlashCard: FlashCard,
   List: List,
   WarningText: WarningText,
+  NationalInsuranceField: TextField,
```

Comment:
just a learner question 
why is there a comma at the end of this property before the `}` within an object?
IntelliJ told me to put a comma too.  I thought this wasn't allowed in JS

Reply:
we have eslint and prettier set up so that we add a dangling comma at the end of objects. It's been valid JS since ES5. dangling/trailing commas are useful for helping with merge conflicts. If you were to add another property at the end and there was no dangling comma, the diff would make it look like you changed  `NationalInsuranceField: TextField` - because you had to add a `,` so it's  
```
NationalInsuranceField: TextField, 
TheNextField: TextField
```
when actually, you didn't change that line at all. 

tldr: it helps with merge conflicts and/or diffs for the reviewer

Reply:
ooh useful to know, thanks :)